### ISSUE TITLE
Build failure with PXR_STRICT_BUILD_MODE - unused template func

### ISSUE THREAD

**Issue by elrond79** [PixarAnimationStudios/USD#1013]
### Description of Issue
If you build with PXR_STRICT_BUILD_MODE on, you get this warning, which is elevated to an error:

```
/mnt/Samsung/projects/rez-build/usd/dev/src/pxr/usd/lib/usd/stage.cpp:5394:6: error: ‘void lpxrInternal__pxrReserved__::{anonymous}::TypedStrongestValueComposer<T>::_ResolveValueImpl(const lpxrInternal__pxrReserved__::PcpNodeRef&, const SdfLayerRefPtr&) [with T = std::map<double, lpxrInternal__pxrReserved__::VtValue>]’ defined but not used [-Werror=unused-function]
 void TypedStrongestValueComposer<SdfTimeSampleMap>::_ResolveValueImpl(
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
```

It looks like this particular TypedStrongestValueComposer<SdfTimeSampleMap> overload is never used... not sure whether you want to remove it completely, comment it out, or do some tricks to make the compiler think it's used / get the warning to go away...

### Steps to Reproduce
1. Build with PXR_STRICT_BUILD_MODE enabled

### System Information (OS, Hardware)
Linux, CentOS 7

### Package Versions
usd-0.19.11.rc3

### Build Flags
PXR_STRICT_BUILD_MODE=1

**Comment by jilliene** [comment]
Filed as internal issue #USD-5663

**Comment by sunyab** [comment]
@elrond79 mentioned in #1021 that this error was fixed in dev late last year, so the fix should be part of the 20.02 release. Closing this out, thanks!
